core/tests/protractor_desktop/topicsAndSkillsDashboard.js

```diff
@@ -117,7 +118,6 @@ describe('Topics and skills dashboard functionality', function() {
     topicsAndSkillsDashboardPage.navigateToUnusedSkillsTab();
     topicsAndSkillsDashboardPage.expectNumberOfSkillsToBe(1);
     topicsAndSkillsDashboardPage.deleteSkillWithIndex(0);
-
```

Comment:
Why delete this line? I think it helps to separate the setup code from the testing code

Reply:
Sorry, i deleted this unknowingly as i was using an extension which automatically removes blank lines. I will correct this.